src/System.Private.CoreLib/shared/System/Diagnostics/Contracts/Contracts.cs

```diff
@@ -180,7 +181,7 @@ public sealed class ContractOptionAttribute : Attribute
         private string _category;
         private string _setting;
         private bool _enabled;
-        private string _value;
```

Comment:
can't put comment up above but for the attributes above (i.e. ContractClassAttribute) there is nothing preventing value from being null (only it doesn't seem to make sense for that to be null) - should there be perhaps some null check there?

Reply:
> only it doesn't seem to make sense for that to be null

Right.  I think we should be comfortable marking things as non-nullable when the intent is for them to not be null / when there's no benefit of them being null, such that a developer gets a warning when they pass in null, even if the implementation doesn't actually validate that the value isn't null.